packages/eslint-plugin-template/src/rules/i18n.ts

```diff
@@ -17,9 +17,9 @@ import {
-const DEFAULT_BOUND_TEXT_ALLOWED_PATTERN = /[a-z]/i;
+const DEFAULT_ALLOWED_BOUND_TEXT_PATTERN = RegExp(`[^${PL_PATTERN}]`);
```

Comment:
It should be correct now. The **allowed** text is everything that is **not** in the alphabet and not the other way around.

Reply:
It should be correct now. The **allowed** text is everything that is **not** in the alphabet (+ latins characters — as expressed by `PL_PATTERN`) and not the other way around.